### DISCUSSION TITLE
Default loss function and modifying loss function

### DISCUSSION THREAD

**Discussion by nirmal129** [lululxvi/deepxde#1239] [Q&A] [answered]
Hello, 

What's the default loss function/metric of DeepXDE NN? And is there any way to provide different loss functions/metrics, such as MAE? If so, how? 

Thank you!

**Comment by lululxvi** [comment, marked as answer]
See https://deepxde.readthedocs.io/en/latest/modules/deepxde.html#deepxde.model.Model.compile
